extras/kms/README.md

```diff
@@ -0,0 +1,88 @@
+# kms package 
```

Comment:
It would be great if this is reviewed for context.

Reply:
LGTM!